src/main/java/oc/wh40k/units/im/IMRetributorSquad.java

```diff
@@ -1,39 +1,44 @@
 package oc.wh40k.units.im;
 
+import static oc.KeyWord.*;
+
 import oc.*;
 
 public class IMRetributorSquad extends Eintrag {
 
     AnzahlPanel squad;
-    OptionsZaehlerGruppe o1;
-    OptionsUpgradeGruppe o2;
-    RuestkammerStarter rkBoss;
-    RuestkammerStarter rkTransport;
+    OptionsZaehlerGruppe bolters;
+    OptionsZaehlerGruppe heavy;
+    OptionsEinzelUpgrade simulacrum;
 
     public IMRetributorSquad() {
+    	super(IMPERIUM, ADEPTUS_MINISTORUM, ADEPTA_SORORITAS, ORDER, INFANTRY, RETRIBUTOR_SQUAD);
         name = "Retributor Squad\n";
         grundkosten = 0;
         überschriftSetzen = true;
 
         squad = new AnzahlPanel(ID, randAbstand, cnt, "Retributor Squad", 5, 10, getPts("Retributor Squad"));
         add(squad);
 
-        add(ico = new oc.Picture("oc/wh40k/images/ASSororitastrupp.jpg"));
-
         seperator();
 
-        ogE.addElement(new OptionsGruppeEintrag("Heavy bolter", getPts("Heavy bolter (AMI)")));
-        ogE.addElement(new OptionsGruppeEintrag("Heavy flamer", getPts("Heavy flamer (AMI)")));
-        ogE.addElement(new OptionsGruppeEintrag("Multi-melta", getPts("Multi-melta (AMI)")));
-        add(o1 = new OptionsZaehlerGruppe(ID, randAbstand, cnt, "", ogE, 4));
-
+        checkBuildaVater();
```

Comment:
Which other Methods needs the BuildaVater?

Reply:
IMAdeptaSororitasRuestkammer.getHeavyWeapons(buildaVater)